tests/transactions/tests.py

```diff
@@ -577,3 +578,94 @@ class DurableTransactionTests(DurableTestsBase, TransactionTestCase):
 
 class DurableTests(DurableTestsBase, TestCase):
     pass
+
+
+@skipUnlessDBFeature("uses_savepoints")
+@skipUnless(connection.supports_async is True, "Async DB test")
+class AsyncTransactionTestCase(TransactionTestCase):
```

Comment:
This fails on postgres with connection pools (`"OPTIONS": {"pool": True}`):
`AttributeError: 'coroutine' object has no attribute 'set_autocommit'`

Reply:
fixed in 9ddddab18b7fe155bfd7fb99f897fbaf2ecce6a6